src/main/java/com/bp3x/raidbot/commands/lfg/LFGCommand.java

```diff
@@ -110,6 +111,9 @@ protected void execute(CommandEvent commandEvent) {
 
                 MessageUtils.autoDeleteMessage(commandEvent.getMessage(), 300);
 
+                ThreadChannel thread = success.createThreadChannel(plannedEvent.getLongName() + "-" + plannedEvent.getEventId()).complete();
```

Comment:
We shouldn't need to block the thread to create the ThreadChannel, then send a message in it. Something like [this](https://github.com/DV8FromTheWorld/JDA/wiki/7\)-Using-RestAction#example-sending-a-private-message) would work better.